R/plots.R

```diff
@@ -62,7 +62,10 @@ plotObservationsCM <- function(...) {
 plotCM <- function(flux.or.conc, show.observations, load.model, finalloads, observations=NULL, 
                    composite=TRUE, linear.interpolation=FALSE, regression=TRUE,  xrange="none", 
                    dateField="Date", verbose=FALSE) {
-  
+  #silly thing needed to pass R CMD check
+  # from http://stackoverflow.com/questions/8096313/no-visible-binding-for-global-variable-note-in-r-cmd-check
+  aes_string=geom_point=yday=xlab=NULL
```

Comment:
same fix here as for `isTimestepRegular`: should be able to put any ggplot2 functions within an `@importFrom ggplot2 ...` line in the roxygen block. `yday` must come from some other package that needs to be imported from in the roxygen, too. `value` and `variable` are the only two that should need to be declared here, and they should please be declared with 
```
value <- variable <- '.ggplot.var'
```
on the line just preceding the ggplot call.